tests/src/common/WskCli.java

```diff
@@ -968,11 +967,9 @@ public RunResult cli(boolean verbose, int expectedExitCode, String... params) th
      * @return RunResult which contains stdout,sterr, exit code
      */
     public RunResult cli(boolean verbose, int expectedExitCode, File workingDir, String... params) throws IllegalArgumentException, IOException {
-        String[] cmd = WhiskProperties.useCliDownload() ? new String[] { getDownloadedCliPath() } : new String[] { WhiskProperties.python, new File(cliDir, binaryName).toString() };
-        String[] args = verbose ? Util.concat(cmd, "--verbose") : cmd;
-        if (binaryName.equals("wsk")) {
```

Comment:
The conditional check is of the binary name is not longer need because wskadmin CLI is not called through this method anymore.